source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -640,14 +671,18 @@ backups tasks. Use during backup to track and montior backup.
   - vm_id: UUID
   - creation_date: TIMESTAMP
 
-Add vm_backup_disk_map table. This table keeps the backup url for every
-disk. This url will be used instead of the image path on the host when
-creating an image transfer for a disk.
+Add vm_backup_disk_map table. This table keeps the backup url and the backup mode
+for every disk. The url will be used instead of the image path on the host when
+creating an image transfer for a disk. The backup mode is used to inform the user
+on the type of backup that was taken for the disk (`full`|`incremental`) so he will
+know what will be the requested context for downloading the backup disk extents
+using imageio API.
 
 - vm_backup_disk_map
   - backup_id: UUID
   - disk_id: UUID
   - backup_url: "nbd:unix:/tmp/<id>.sock:exportname=<sdb>" | "nbd://localhost:<12345>/<sdb>"
```

Comment:
please keep value of backup_url key in `` or html will fail with invalid element <sbd> , <12345>, <id>

Reply:
Not related to this patch, I will fix it in a different patch